src/app/pages/userManagment/user-management/user-management.html

```
@@ -83,79 +91,64 @@
+                            'bg-red-800':user.status=='Deleted',
                                 'bg-yellow-500':user.status=='PendingVerification',
-                                'bg-blue-400':user.status=='EmailVerified'
+                                'bg-blue-400':user.status=='EmailVerified',
+                                'bg-red-400':user.status=='Blacklisted'
                             }" class="h-2.5 w-2.5 rounded-full  me-2"></div> {{user.status}}
                         </div>
                     </td>
                     <td class="px-6 py-4">
-                        <a type="button" class="font-medium text-blue-600  hover:underline" (click)="showUpdateUser()">Edit user</a>
+                        <select
+                        id="statusFilter"
```

HTMLHint (1.5.0)

[error] The id value [ statusFilter ] must be unique.

(id-unique)